source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -768,7 +782,8 @@ namespace RendererQml
         auto backgroundTag = std::make_shared<QmlTag>("Rectangle");
         backgroundTag->Property("radius", "5");
         //TODO: These color styling should come from css
-        backgroundTag->Property("color", Formatter() << input->GetId() << ".hovered ? 'lightgray' : 'white'");
+        //TODO: ADD hover effect
+        backgroundTag->Property("color", "'transparent'");
```

Comment:
Setting the background as transparent will make the highlight glow visible inside the rectangle as well
![image](https://user-images.githubusercontent.com/78541663/110316654-09236700-8031-11eb-9907-7183c64ac914.png)


We can set the color as the base card's color, ( adaptiveCard.color ) to make the glow visible only on the outside
![image](https://user-images.githubusercontent.com/78541663/110316250-75519b00-8030-11eb-8513-c496b81be036.png)

If I'm not wrong similar behavior will be observed for the other input elements as well

Reply:
let's use `defaultPalette.backgroundColor` instead of `transparent` where required.

sorry for the confusion.